templates/ui/utils/scrollable/Scrollable.d.ts

```diff
@@ -107,4 +107,6 @@ declare class Scrollable extends Sizer {
     setMouseWheelScrollerEnable(enable?: boolean): this;
 
     readonly scrollMode: number;
-}
\ No newline at end of file
+
+    readonly isOverflow: boolean;
+}
```

Comment:
The isOverflow property is described in the [documentation](https://rexrainbow.github.io/phaser3-rex-notes/docs/site/ui-scrollablepanel/?h=scrollable#scroll-content), but was not described in the types.

Property is inherited from [Scrollable](https://github.com/rexrainbow/phaser3-rex-notes/blob/bdfd8834852b53e072d85bec74a713999374b7ef/templates/ui/utils/scrollable/Scrollable.js#L173).